### PR TITLE
Workflow catalog: send workflows to a remote scheduler and add action combo box

### PR DESCRIPTION
**Pull request by laurianed**

### CODE REVIEW

File: app/scripts/proactive/view/CatalogView.js [hunk 1]
```diff
@@ -30,10 +30,13 @@ define(
         internalSwitchBucket: function (currentBucketID) {
             this.$('#catalog-workflow-list').empty();
             var emptyView = _.template(catalogEmpty);
+            var publishButton = $('#publish-to-catalog-button');
+            var StudioApp = require('StudioApp');
```
Comment (MaelAudren) [inline app/scripts/proactive/view/CatalogView.js]: Change "StudioApp" for "studioApp"

File: app/scripts/proactive/view/dom.js [hunk 2]
```diff
@@ -373,18 +367,9 @@ define(
...
-        
-        $("#import-archive-button").click(function (event) {
-            event.preventDefault();
-            var studioApp = require('StudioApp');
-            closeCollapsedMenu();
-            $("#import-archive-file").parent('form').trigger('reset');
-            $("#import-archive-file").click();
-        })
+            var selectedBucketId = bucketId ? bucketId : $("#select-bucket").val();
+            return '/workflow-catalog/buckets/' + selectedBucketId + '/workflows/' + wfIds + '?alt=zip';
```
Comment (MaelAudren) [inline app/scripts/proactive/view/dom.js]: Zip files are currently not supported by the workflow-catalog so the url aim to a 415 error Unsupported Media Type. Is it intended ?

File: app/scripts/proactive/view/dom.js [hunk 3]
```diff
@@ -373,18 +367,9 @@ define(
...
+        }
         
         $("#import-archive-file").change(function (event) {
         	var StudioApp = require('StudioApp');
```
Comment (MaelAudren) [inline app/scripts/proactive/view/dom.js]: Replace "StudioApp" by "studioApp"

File: app/scripts/proactive/view/dom.js [hunk 4]
```diff
@@ -459,6 +397,45 @@ define(
...
+			};
+			xhr.send(null);
+        });
+        
+        function send_archive_to_catalog(blob, remoteURL, bucketId){
+            var payload = new FormData();
+            payload.append('file', blob);
+            var selectedBucketId = bucketId ? bucketId : $("#select-bucket").val();
+            var server = remoteURL ? remoteURL : "";
+            var url = server + '/workflow-catalog/buckets/' + selectedBucketId + '/workflows?alt=zip';
```
Comment (MaelAudren) [inline app/scripts/proactive/view/dom.js]: Zip files are not supported currently by the catalog.

File: app/index.html [hunk 5]
```diff
@@ -371,6 +371,25 @@
...
+        <div id="send-to-remote-confirmation-modal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                        <h3>Send selected Workflows to a remote scheduler</h3>
+                    </div>
+                    <div class="modal-body" style="height:100%">
+                        <table style="width:100%"><tr><td>Please enter the remote Scheduler<br>URL (http://xxx.xxx.xxx.xxx:port)</td><td><input type="text" id="remoteCatalogURL"></td></tr>
+                        <tr><td>Please enter the id of the bucket<br>of the remote scheduler</td><td><input type="text" id="remoteCatalogBucketId"></td></tr></table>
```
Comment (sophiesongge) [inline app/index.html]: This sentence is a little bit strange to me, if I know nothing about workflow-catalog.
Is it better to change the second "of" to "on"?

**Pull request merged**